lib/exnumerator/keyword_list_enum.ex

```diff
@@ -0,0 +1,34 @@
+defmodule Exnumerator.KeywordListEnum do
+  def cast(values, term) do
+    atom_term = if is_binary(term), do: String.to_atom(term), else: term
```

Comment:
Maybe it will be worth to introduce `guard`ed function here?

Reply:
You can't,

Line 8 uses the term instead of the atom one.